test/unit/services/sshAdapter_spec.js

```diff
@@ -0,0 +1,285 @@
+"use strict";
+/*jslint plusplus: true */
```

Comment:
Is this really necessary? Seems like either this is wrong or the style rule is wrong.